### DISCUSSION TITLE
Typer Cheat Sheet

### DISCUSSION THREAD

**Discussion by harkabeeparolus** [fastapi/typer#634] [Show and tell]
I got a little tired of flipping through the many pages of the very good Typer Tutorial, so I made a one-page summary:

* [Typer Cheat Sheet](https://gist.github.com/harkabeeparolus/a6e18b1f4f4f938f450090c5e7523f68)

I think that the Tutorial is excellent, and contains a ton of great extra information. But I was missing some kind of reference docs. For instance, Click has the [full API reference of all public classes and functions](https://click.palletsprojects.com/en/8.1.x/api/) which is pretty great. I wish Typer had something similar, but at least this is a start! 😄

I welcome all constructive feedback. 😊

**Comment by Apocryphon-X** [comment]
This is really good! I don't use Typer so much because of #607 but even then I find this tutorial really useful and interesting. Thanks for this contribution, I hope more people gets to see this post :)

**Comment by svlandeg** [comment]
This is awesome, thanks so much for sharing! I might in fact tweet about this to hopefully give it more visibility in the community 😇
